mongodm.go

```diff
@@ -311,19 +308,16 @@ func (self *Connection) Open() (err error) {
 		return err
 	}
 
-	self.session = session
-
-	self.session.SetMode(mgo.Monotonic, true)
-
-	self.database = self.session.DB(self.Config.DatabaseName)
```

Comment:
`self.database` is only used on line 270 which has been replaced with `self.Session.DB("")`